src/bcftools/bcftools_norm/config.vsh.yaml

```
@@ -0,0 +1,187 @@
+name: bcftools_norm
+namespace: bcftools
+description: | 
+  Left-align and normalize indels, check if REF alleles match the reference, split multiallelic sites into multiple rows; 
+  recover multiallelics from multiple rows. 
+keywords: [Annotate, VCF, BCF]
```

perhaps "Normalize" instead of "Annotate"?
```suggestion
keywords: [Normalize, VCF, BCF]
```